Sources/Authentication/Helpers/NetworkHelper.swift

```diff
@@ -39,15 +39,17 @@ extension AuthenticationHandler {
 
 extension AuthenticationHandler {
     func createAuthorizationURL() -> URL? {
+        guard let configuration = configuration else { return nil }
+
         let queryItems = [
           URLQueryItem(name: "client_id", value: configuration.clientID),
-          URLQueryItem(name: "redirect_uri", value: self.configuration.callBackURL),
+          URLQueryItem(name: "redirect_uri", value: configuration.callBackURL),
           URLQueryItem(name: "response_type", value: "code"),
-          URLQueryItem(name: "scope", value: self.configuration.scopes.joined(separator: " ")),
+          URLQueryItem(name: "scope", value: configuration.scopes.joined(separator: " ")),
           URLQueryItem(name: "code_challenge_method", value: "S256"),
           URLQueryItem(name: "code_challenge", value: configuration.codeChallenge)
         ]
-        guard let accessTokenURL = URL(string: self.configuration.baseURL + self.configuration.authorizePath) else { return nil }
+        guard let accessTokenURL = URL(string: configuration.baseURL + configuration.authorizePath) else { return nil }
```

Comment:
You could move this guard "accessTokenURL" up to "guard let configuration = configuration else { return nil }" and do both in a single guard